tests/wfc3/test_uvis_12single.py

```diff
@@ -27,6 +27,8 @@ def _single_raw_calib(self, rootname):
 
     # Ported from ``calwf3_uv_12``.
     @pytest.mark.parametrize(
+        'rootname', ['iaao09l0q'])
+        """
```

Comment:
I'd rather this is commented out than being turned into a docstring. Same comment applies to all other similar occurrences.

Reply:
Understood.  I was saving myself some typing.  I actually want to remove many of these datasets, but I would rather do the actual removal when I have time to make sure I am absolutely not deleting something that is useful.

Reply:
Fixed.